presto-docs/src/main/sphinx/security/file-based-system-access-control.rst

```diff
@@ -0,0 +1,93 @@
+================================
+File Based System Access Control
+================================
+
+Presto offers a file based ``SystemAccessControl`` plugin that supports
+specifying access control rules using JSON in a file. In order to enable the
```

Comment:
wait to mention JSON lower down when you specify the format, just mention a file up here